### PR TITLE
Take care of camera zoom in debug mode

### LINKED ISSUE #2632: Draw collisions hitboxes and points not following on layers
If you have layers that follow an object and use the action 'Draw collisions hitboxes and points' it doesn't follow for those layers.

Screenshot 2021-05-10 214159

GDevelop version: 109
OS: Windows 10

### PR DESCRIPTION
**Pull request by Bouh**
Close #2632

### CODE REVIEW

**Comment by Bouh** [comment]
Wait two minutes, i've seen the rotation on camera wasn't correct too 😬

**Comment by 4ian** [comment]
At this point, we might as well use `layer.convertInverseCoords(x,y)` to calculate the position of each point of the boxes you're drawing. This takes as an argument a position in the "scene space", will take into account the rotation/zoom/position of the layer, and will give you back the position that corresponds to it relative to the canvas (i.e: a position that you can use directly in a PIXI.Container that is at the origin).

In other words, something like: 
```
const topLeft = layer.convertInverseCoords(aabb.min[0], aabb.min[1]);
const bottomRight = layer.convertInverseCoords(aabb.max[0], aabb.max[1]);
debugDraw.drawRect(topLeft[0], topLeft[1], bottomRight[0] - topLeft[0], bottomRight[1] - topLeft[1]);
```

(this is untested).

Note that this approach is a lot of computations for all objects. It's ok because this is a debug view, not intended for performance (to improve performance, we would put the boxes directly inside a container that has the same transform as the layer, or inside the layer container... but it's starting to get complex. Let's keep things simple)

File: GDJS/Runtime/pixi-renderers/runtimescene-pixi-renderer.ts [hunk 1]
```diff
@@ -149,21 +149,31 @@ namespace gdjs {
...
+        const vertices = [
+          new PIXI.Point(aabb.min[0], aabb.min[1]),
+          new PIXI.Point(aabb.max[0], aabb.min[1]),
+          new PIXI.Point(aabb.max[0], aabb.max[1]),
+          new PIXI.Point(aabb.min[0], aabb.max[1]),
+        ];
+
+        const polygon: float[] = [];
+        for (let v = 0; v < vertices.length; v++) {
+          let point = layer.convertInverseCoords(vertices[v].x, vertices[v].y);
```
[the remark this answers is not in the document]
Comment (Bouh) [reply GDJS/Runtime/pixi-renderers/runtimescene-pixi-renderer.ts]: Fixed

**Comment by 4ian** [comment]
Thanks! I did a quick optimisation. I'll release a version with this fixed and the bug for collision masks/points editor.

**Comment by Bouh** [comment]
Got it thanks!

**Pull request merged**